tracing/src/main/java/com/palantir/tracing/DetachedSpan.java

```diff
@@ -19,13 +19,33 @@
 import com.google.errorprone.annotations.MustBeClosed;
 import com.palantir.tracing.api.SpanType;
 import java.io.Closeable;
+import javax.annotation.CheckReturnValue;
 
 /**
  * Span operation which is not bound to thread state, and can measure operations which
  * themselves aren't bound to individual threads.
  */
 public interface DetachedSpan extends Closeable {
 
+    /**
+     * Like {@link Tracer#startSpan(String, SpanType)}, but does not set or modify tracing thread state.
+     * Creates a detached child span using the callers current span as a parent, if it is present, otherwise
+     * creates a detached root span with a new traceId.
+     */
+    @CheckReturnValue
+    static DetachedSpan start(String operation, SpanType type) {
+        return Tracer.detachInternal(operation, type);
+    }
+
+    /**
+     * Opens a new {@link SpanType#LOCAL LOCAL} detached span for this thread's call trace,
+     * labeled with the provided operation.
+     */
+    @CheckReturnValue
+    static DetachedSpan start(String operation) {
```

Comment:
@ellisjoe @ferozco I've renamed these from `detach` to `start`, otherwise method names conflict with non-static methods. I think `DetachedSpan.start` conveys the detached nature of the span as well as `Tracer.detach`.